2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Configure stats pid to 6635
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Loading settings from /home/kjk/.config/wandb/settings
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Loading settings from /home/kjk/Documents/Workspace/NeuralNetfromScratch/src/models/wandb/settings
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_init.py:_log_setup():526] Logging user logs to /home/kjk/Documents/Workspace/NeuralNetfromScratch/src/models/wandb/run-20240221_112222-t8ldsynw/logs/debug.log
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_init.py:_log_setup():527] Logging internal logs to /home/kjk/Documents/Workspace/NeuralNetfromScratch/src/models/wandb/run-20240221_112222-t8ldsynw/logs/debug-internal.log
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_init.py:init():566] calling init triggers
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-21 11:22:22,719 INFO    MainThread:6635 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 65xoq4zd
2024-02-21 11:22:22,720 INFO    MainThread:6635 [wandb_run.py:_finish():1970] finishing run kimkj/mnist-model-evaluation/65xoq4zd
2024-02-21 11:22:22,720 INFO    MainThread:6635 [jupyter.py:save_history():445] not saving jupyter history
2024-02-21 11:22:22,720 INFO    MainThread:6635 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-21 11:22:22,720 INFO    MainThread:6635 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-21 11:22:22,721 INFO    MainThread:6635 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-21 11:22:22,721 INFO    MainThread:6635 [wandb_run.py:_restore():2193] restore
2024-02-21 11:22:22,721 INFO    MainThread:6635 [wandb_run.py:_restore():2199] restore done
2024-02-21 11:22:34,330 INFO    MainThread:6635 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-21 11:22:34,330 INFO    MainThread:6635 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-21 11:22:34,335 INFO    MainThread:6635 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-21 11:22:34,450 INFO    MainThread:6635 [wandb_init.py:init():616] starting backend
2024-02-21 11:22:34,450 INFO    MainThread:6635 [wandb_init.py:init():620] setting up manager
2024-02-21 11:22:34,454 INFO    MainThread:6635 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-21 11:22:34,454 INFO    MainThread:6635 [wandb_init.py:init():628] backend started and connected
2024-02-21 11:22:34,461 INFO    MainThread:6635 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-21 11:22:34,461 INFO    MainThread:6635 [wandb_init.py:init():720] updated telemetry
2024-02-21 11:22:34,464 INFO    MainThread:6635 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-21 11:22:35,694 INFO    MainThread:6635 [wandb_run.py:_on_init():2262] communicating current version
2024-02-21 11:22:35,798 INFO    MainThread:6635 [wandb_run.py:_on_init():2271] got version response 
2024-02-21 11:22:35,799 INFO    MainThread:6635 [wandb_init.py:init():804] starting run threads in backend
2024-02-21 11:22:35,832 INFO    MainThread:6635 [wandb_run.py:_console_start():2241] atexit reg
2024-02-21 11:22:35,832 INFO    MainThread:6635 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-21 11:22:35,832 INFO    MainThread:6635 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-21 11:22:35,832 INFO    MainThread:6635 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-21 11:22:35,833 INFO    MainThread:6635 [wandb_init.py:init():847] run started, returning control to user process
2024-02-21 11:25:54,710 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Configure stats pid to 6635
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Loading settings from /home/kjk/.config/wandb/settings
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Loading settings from /home/kjk/Documents/Workspace/NeuralNetfromScratch/src/models/wandb/settings
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_init.py:_log_setup():526] Logging user logs to /home/kjk/Documents/Workspace/NeuralNetfromScratch/src/models/wandb/run-20240221_112554-x3y0m7p2/logs/debug.log
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_init.py:_log_setup():527] Logging internal logs to /home/kjk/Documents/Workspace/NeuralNetfromScratch/src/models/wandb/run-20240221_112554-x3y0m7p2/logs/debug-internal.log
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fe4943188e0>
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_init.py:init():566] calling init triggers
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-21 11:25:54,711 INFO    MainThread:6635 [wandb_init.py:init():591] re-initializing run, found existing run on stack: t8ldsynw
2024-02-21 11:25:54,712 INFO    MainThread:6635 [wandb_run.py:_finish():1970] finishing run kimkj/mnist-model-evaluation/t8ldsynw
2024-02-21 11:25:54,712 INFO    MainThread:6635 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-21 11:25:54,712 INFO    MainThread:6635 [wandb_run.py:_restore():2193] restore
2024-02-21 11:25:54,712 INFO    MainThread:6635 [wandb_run.py:_restore():2199] restore done
2024-02-21 11:25:59,891 INFO    MainThread:6635 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-21 11:25:59,891 INFO    MainThread:6635 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-21 11:25:59,897 INFO    MainThread:6635 [wandb_run.py:_footer_sync_info():3825] logging synced files
